nss-day-cohort-41/react-nutshell-the-macadamias
# Messages add friend

# Description
Added functionality to add a new friend from the messages list. Updated all mentions of activeUserId to just id when retrieving the user id from session storage. Made some simple css styling to the messages display and input field.

## Type of change

# Testing Instructions
Make sure there are users who are not friends with the active user in the database and have written a message. An 'add' button should be visible in the non-friend user's message card. When 'add' is clicked, a modal should appear with a message to confirm if active user wants to add them as a new friend. Once 'Yes' is clicked, the active user should be redirected back to the main messages and should see the 'add' button gone for the new friend's messages. Additionally, this new friend should be populated in the friends section on the database.

# Checklist:
